### PR TITLE
Add "excessive diskspace" AT eligibility hold

### PR DESCRIPTION
**Pull request by lamosty**
@jmdodd added another AT eligibility hold for excessive diskspace sites (> 50 gigs). In this PR, we are adding it to Calypso:

image

## Testing
Follow instructions in D6373-code.

### CODE REVIEW

**Comment by kwight** [comment]
Just noting "diskspace" should be two words: "disk space".

File: client/blocks/eligibility-warnings/hold-list.jsx [hunk 1]
```diff
@@ -71,7 +71,13 @@ function getHoldMessages( siteSlug, translate ) {
 			description: translate(
 				'Please check your email to confirm your address.'
 			),
-		}
+		},
+		EXCESSIVE_DISK_SPACE: {
+			title: translate( 'Excessive diskspace' ),
+			description: translate(
+				'This site cannot be transferred due to diskspace limitations.'
```
Comment (gwwar) [inline client/blocks/eligibility-warnings/hold-list.jsx]: I'm sure Editorial has better ideas, but how about: "Site Too Large: Media usage must be less than 50GB"

Comment (kwight) [inline client/blocks/eligibility-warnings/hold-list.jsx]: I think that's also confusing, if your site already has 350GB and is working fine before trying to install the plugin :/

Comment (gwwar) [inline client/blocks/eligibility-warnings/hold-list.jsx]: There's nothing really reasonable we can ask them to do here. Hopefully we'll be able to remove this in the future.

**Comment by kwight** [comment]
Aside from copy concerns, I tested to instructions in D6373-code and all's good 🚢 👍

**Comment by gwwar** [comment]
Whoops, probs should have done a merge over squash, but hope you don't mind @lamosty

**Pull request merged**